crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -116,6 +116,17 @@ def repocheck
       end
     end
 
+    def ceph_healthy?
+      ceph_node = NodeObject.find("roles:ceph-mon AND ceph_config_environment:*").first
+      return true if ceph_node.nil?
+      ssh_retval = ceph_node.run_ssh_cmd("ceph health 2>&1")
```

Comment:
Maybe we should also check output from ceph health, I would propose something like this

```
ssh_retval = ceph_node.run_ssh_cmd("ceph health | grep -q HEALTH_OK")
```

What do you think?